include/extractor/extraction_way.hpp

```diff
@@ -96,6 +96,7 @@ struct ExtractionWay
     bool roundabout;
     bool circular;
     bool is_startpoint;
+    bool ignore_in_grid;
```

Comment:
yes this has to go - not only here but also in the profiles and the bindings, see https://github.com/Project-OSRM/osrm-backend/pull/3382#pullrequestreview-11385147